Library/Formula/proj.rb

```diff
@@ -86,4 +86,9 @@ def install
                           "--prefix=#{prefix}"
     system "make", "install"
   end
+  test do
+    system "proj -l"
+    system "echo -120 36 | proj +proj=tmerc +ellps=WGS84 +units=km +lon_0=-121.15 +lat_0=34.15"
```

Comment:
`assert_match "something", pipe_output("proj +proj=tmerc +ellps=WGS84 +units=km +lon_0=-121.15 +lat_0=34.15", "-120 36", 0)` and you can also scrap the test above and below if you like too.